tests/test_hinfo.py

```diff
@@ -1,51 +1,40 @@
 import os
-import shutil
-import sqlite3
-import tempfile
 from pathlib import Path
 
+import lsst.utils
 import pytest
+import sqlalchemy as sa
 import yaml
-from lsst.consdb import hinfo, utils
+from felis.datamodel import Schema
+from felis.db.utils import DatabaseContext
+from felis.metadata import MetaDataBuilder
+from felis.tests.postgresql import setup_postgres_test_db
+from lsst.consdb import hinfo
 from lsst.resources import ResourcePath
-from sqlalchemy import MetaData, Table, select
+from sqlalchemy import MetaData, select
 
 
 @pytest.fixture
-def tmpdir(scope="module"):
-    os.environ["INSTRUMENT"] = "LATISS"
-    tmpdir = Path(tempfile.mkdtemp())
-    try:
-        yield tmpdir
-    finally:
-        shutil.rmtree(tmpdir)
+def pg_engine(request, scope="module"):
+    schema_name = request.param if hasattr(request, "param") else "cdb_latiss"
 
+    schema_file = os.path.join(lsst.utils.getPackageDir("sdm_schemas"), "yml", schema_name + ".yaml")
+
+    with open(schema_file) as f:
+        yaml_data = yaml.safe_load(f)
+    schema = Schema.model_validate(yaml_data)
+    md = MetaDataBuilder(schema).build()
+
+    with setup_postgres_test_db() as instance:
+        context = DatabaseContext(md, instance.engine)
+        print(f"{type(instance.engine)=}")
```

Comment:
We generally shouldn't have "debugging" `print()` calls in the code.  But maybe there's a good reason for it, as I see one in `test_pqserver.py` that seems to be left in despite a comment.

Reply:
No, it's just an oversight.